tests/Unit/API/Site/Controllers/MerchantCenter/AccountReviewControllerTest.php

```diff
@@ -22,19 +18,18 @@ class AccountReviewControllerTest extends RESTControllerUnitTest {
 	public function setUp() {
 		parent::setUp();
 
-		$this->request_review = $this->createMock( RequestReview::class );
-		$this->controller     = new AccountReviewController( $this->server, $this->request_review );
+		$this->controller     = new AccountReviewController( $this->server );
 		$this->controller->register();
 	}
 
 	public function test_route() {
-		$this->request_review->method( 'get_next_attempt' )
-		                     ->willReturn( mktime( 0, 0, 0 ) );
-
 		$response = $this->do_request( self::ROUTE_GET_REQUEST );
 
 		$this->assertEquals( 200, $response->get_status() );
-		$this->assertEquals( mktime( 0, 0, 0 ), $response->get_data()['nextAttempt'] );
+
+		$response_data = $response->get_data();
+		$this->assertTrue( isset( $response_data['status'] ) );
```

Comment:
❔

I guess that these tests are temporary until the Google API is ready so then we can test different scenario.

Reply:
Yes, this is just for now temporary... unfortunately we cannot test much more right now in the Backend part, since the response is hardcoded, so I'm naily testing the route and the properties.